docs/deploy-hyperlane.mdx

```diff
@@ -1,113 +1,334 @@
 
-We've partnered with [Kurtosis](https://www.kurtosis.com/) to provide a single-click validator and relayer deployment in the cloud. You'll receive a one-month trial during which your validator and relayer run in Kurtosis Cloud for free. After your trial ends, you'll need to either run your validator and relayer yourself or purchase a Kurtosis Cloud subscription.
+```tsx
+Skipping updating chain yourchain at github registry
+Now updating chain yourchain at filesystem registry at $HOME/.hyperlane
+Done updating chain yourchain at filesystem registry
+```
 
-To get started, use the `deploy kurtosis-agents` command to generate a link for configuring your Kurtosis deployment.
+Now, under `$HOME/.hyperlane/chains` you will find a new folder named with your custom chain’s name, and a file named `metadata.yaml` within that folder.
+
+<details> 
+  <summary>
+    Example content of this config at `$HOME/.hyperlane/chains/yourchain/metadata.yaml`
+  </summary>
+
+   ```yaml
+    # yaml-language-server: $schema=../schema.json
+    name: yourchain
+    chainId: 171717
+    domainId: 171717
+    protocol: ethereum
+    rpcUrls:
+      - http: https://hyper-lane-docs.rpc.caldera.xyz/http
+    nativeToken:
+      name: Ether
+      symbol: ETH
+      decimals: 18
+    ```
+</details>
+
+## 3) Core: configure & deploy your custom chain’s core contracts
+
+Next, let’s configure, deploy and test your custom chain’s core contracts.
+
+### Initialize configuration
+
+1. From your local environment, source the private key or seed phrase of your funded deployer address via `HYP_KEY`. For example: `export HYP_KEY='<YOUR_PRIVATE_KEY>'`
+2. Verify your `HYP_KEY` is sourced correctly via `echo $HYP_KEY`
+3. From the same instance, run:
+    
+  ```bash
+  hyperlane core init
+  ```
+<details>
+  <summary>
+    After successfully creating a new core deployment config, you can find it in a file named `core-config.yaml` within  `./configs`
+  </summary>
+  
+  ```yaml
+  owner: "0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb"
+  defaultIsm:
+    type: trustedRelayerIsm
+    relayer: "0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb"
+  defaultHook:
+    type: merkleTreeHook
+  requiredHook:
+    owner: "0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb"
+    type: protocolFee
+    beneficiary: "0x16F4898F47c085C41d7Cc6b1dc72B91EA617dcBb"
+    maxProtocolFee: "100000000000000000"
+    protocolFee: "0"
+  ```
+    
+</details>
+
+
+### Deploy contracts
+
+Continuing in the same instance with your `HYP_KEY` sourced, run:
 
 ```bash
-hyperlane deploy kurtosis-agents
+hyperlane core deploy
 ```
 
-Follow the prompts and then click on the Kurtosis Cloud link output by the command.
-
-If this is your first time using Kurtosis Cloud, you'll need to sign in with Google and Kurtosis will provision a remote cloud instance for your validator and relayer in roughly 2 minutes.
+Use the arrows and enter to select your custom chain from the bottom of the mainnet list. 
```

Comment:
'from below the mainnet list'. Though tbh it would be a UX win to put custom chains first. I thought we did that but I guess not!